utils/checks.stanza

```diff
@@ -1728,23 +1768,25 @@ public defn propagate-power-states (module:JITXObject) :
   ; DFS walk of the netlist, to update power-states of 
   ; connected power-pins 
   val visited = HashSet<JITXObject>()
-  defn visit (pin:JITXObject, power-states:Tuple<PowerState>) :
+  defn visit-pins (pin:JITXObject, power-states-tuple:Tuple<PowerState>) :
```

Comment:
Style note: since `power-states` has the `Tuple` as an annotation already it doesn't need it to be annotated in the variable name